tests/integration_tests/ui_tests/ArelleGUITest/ArelleGUITest/Tests.cs

```diff
@@ -0,0 +1,373 @@
+                () => Window.FindFirstChild(Automation.ConditionFactory.ByControlType(ControlType.Window)), timeout: TimeSpan.FromSeconds(3)).Result;
+            var selectFileFromArchivePane = selectFileFromArchiveWindow.FindFirstByXPath("/Pane/Pane/Pane/Pane[1]");
+            Mouse.LeftClick(new Point(
+                (int)(selectFileFromArchivePane.BoundingRectangle.X + selectFileFromArchivePane.BoundingRectangle.Width / 2.0f),
+                (int)(selectFileFromArchivePane.BoundingRectangle.Y
+                // +1 for the header, +0.5 to end up in the middle
+                + selectFileFromArchivePane.BoundingRectangle.Height * (indexOfEntryPoint + 1.5) / (nFilesInDocumentZip + 1))
+            ));
+            Keyboard.Type(VirtualKeyShort.TAB, VirtualKeyShort.ENTER);
+
+            Logger.Default.Info("wait for document to load and UI to populate");
+            Retry.WhileNull(() => Window.FindFirstByXPath("/Pane/Pane/Pane/Pane/Pane/Pane/Pane"), timeout: TimeSpan.FromMinutes(1), ignoreException: true);
+            var iconsAndMainArea = Window.FindFirstByXPath("/Pane/Pane");
+            var iconsAndMainAreaChildren = iconsAndMainArea.FindAllChildren();
+            var mainArea = iconsAndMainAreaChildren[0];
+            var icons = iconsAndMainAreaChildren[1];
+            var statusBar = iconsAndMainAreaChildren[2];
+            var mainAreaChildren = mainArea.FindAllChildren();
+            var messagesAreaAndTabBar = mainAreaChildren[0];
+            var tablesAndFactView = mainAreaChildren[1];
+            var messagesArea = messagesAreaAndTabBar.FindFirstChild();
+            var tablesAndFactViewChildren = tablesAndFactView.FindAllChildren();
+            var factViewAndTabBar = tablesAndFactViewChildren[0];
+            var tablesAndTabBar = tablesAndFactViewChildren[1];
+            var factView = factViewAndTabBar.FindFirstChild();
+            var tables = tablesAndTabBar.FindFirstChild();
+
+            Logger.Default.Info("copy structure to clipboard");
+            Mouse.RightClick(tables.BoundingRectangle.Center());
+            Wait.UntilInputIsProcessed();
+            var copyToClipboardMenuItem = Window.ContextMenu.Items[4];
+            copyToClipboardMenuItem.Click();
+            Wait.UntilInputIsProcessed();
+            var copyToClipboardSubMenu = Automation.GetDesktop()
+                .FindAllChildren(cf => cf.ByControlType(ControlType.Menu).And(cf.ByProcessId(Application.ProcessId)))
+                .Select(e => e.AsMenu())
+                .Single(e => e.Items.Length == 3);
+            var copyTableMenuItem = copyToClipboardSubMenu.Items[2];
+            copyTableMenuItem.Click();
+            Wait.UntilInputIsProcessed();
+            Thread.Sleep(200);
+            Logger.Default.Info("get clipboard text");
+            var tableText = GetClipboardText();
+            Assert.That(tableText, Is.EqualTo(TABLE_STRUCTURE.Replace("\r", "")));

```

Comment:
This should probably either be screen recording the entire time as well as providing a screenshot when it fails?